foxi/onnxifi.h

```diff
@@ -863,6 +863,22 @@ typedef struct onnxTensorDescriptorV1 {
    * For a scalar, this pointer can be NULL.
    */
   const uint64_t* shape;
+  /**
+   * A flag indicates whether this tensor is a quantized tensor or not.
+   * If it is a quantized tensor, the value should be set to 1.
+   * By default it should be set to 0.
+   */
+  unsigned char is_quantized;
```

Comment:
why not bool or uint8_t?

Reply:
C?

Reply:
Cause it is C?

Reply:
we already use int32_t anyway, why bother using uint8_t?

Reply:
That's true.

Reply:
Then let me try uint8_t? didnt see the difference...

Reply:
it's weird to use char to store the number, which means you need explict conversion. So if it's possible, that's use uint8 instead

Reply:
ok, let me change it.